content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

```diff
@@ -0,0 +1,201 @@
+===========================================
+Set up Starshipit shipping services in Odoo
+===========================================
+
+Starshipit is a shipping service operator that facilitates the integration of Australasian
+shipping carriers with Odoo. Once integrated, users can create shipping methods that will
+automatically get rates from specific couriers (such as Australia Post, NZ Post, DHL,...)
+based on predefined conditions.
+
+Setup in Starshipit
+===================
+
+Create an account and activate carriers
+---------------------------------------
+
+To get started, go to `Starshipit's platform <https://starshipit.com/>`_ to configure the account
+and generate the connector credentials. Log in with the Starshipit account, or create a new one if
+needed.
+
+Pickup address configuration
+----------------------------
+
+Once logged into the Starshipit account, navigate to :menuselection:`Settings --> Pickup address`,
+and fill in the field for :guilabel:`Pickup address`. It should be the same address as the
+warehouse you are shipping from.
+
+.. image:: starshipit_shipping/starshipit-settings-address.png
+   :align: center
+   :alt: Adding addresses in the Starshipit settings.
+
+Couriers configuration
+----------------------
+
+Next, navigate to :menuselection:`Settings --> Couriers`,
+and add the :guilabel:`Couriers` you wish to integrate with.
+
+.. image:: starshipit_shipping/starshipit-settings-couriers.png
+   :align: center
+   :alt: Adding addresses in the Starshipit settings.
+
+.. tip::
+   You can find details on how to integrate with the different couriers in
+   `Starshipit's support center <https://support.starshipit.com/hc/en-us/>`_.
+
+Couriers services configuration
+-------------------------------
+
+In the checkout rates settings, you will need to set which courier services you want to provide in
+Odoo.
+
+.. image:: starshipit_shipping/starshipit-checkout-rate.png
+   :align: center
+   :alt: Checkout rates in the Starshipit settings.
+
+Shipping rules configuration
+----------------------------
+
+Finally, you will need to configure shipping rules in Starshipit. These are used when placing a
+shipping order to set the correct courier on your order. There are multiple ways to do so. We
+recommend adding a rule with the condition being ":guilabel:`contains`", the value being
+":guilabel:`the product code`", and the action "setting that product code and courier" to the
+shipping order.
```

Comment:
I wanted to help add some additional details about what a shipping rule does and how it's used, for beginners who don't really understand why this is necessary (:
```suggestion
Configure shipping rules to assign the correct shipping methods to orders based on specific
conditions.

To create a rule, go to :menuselection:`Settings --> Rules` and click :guilabel:`Add a new rule`.

While there are multiple ways to configure rules, it is recommended to set:

#. :guilabel:`Condition` to :guilabel:`Contains`
#. :guilabel:`Value` to the `product code`
#. :guilabel:`Action` to :guilabel:`Set Courier & Product Code`
```